.github/workflows/merge.yml

```diff
@@ -410,6 +410,27 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 2
 
+      - name: Validate downloaded file after saved changes for Windows
+        if: matrix.os == 'windows-latest'
```

Comment:
I dont think this need only be windows.

Here with run, we can set the `shell` to be `bash` and then write something that will test across all platforms and surface any inconsistency there.

Reply:
Okay. Test operations for Linux and Windows have different dependencies. I can write tests to cover both.